Component/Button/button.css

```diff
@@ -0,0 +1,198 @@
+@import url("../../Utility CSS/navbar.css");
+.box {
+  height: 14rem;
+  width: 120rem;
+  justify-content: flex-start;
+  padding: 2rem;
+}
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color:var(--btn-primary-color
+  background-color: none;
+}
+
+.btn:hover,
+.btn-link:hover {
+  filter: brightness(0.9);
+  cursor: pointer;
+}
+
+.btn-primary {
+  background-color: var(--btn-primary-color);
+}
+
+.btn-secondary {
+  background-color: var(--btn-secondary-color);
+}
+
+.btn-success {
+  background-color: var(--btn-success-color);
+}
+
+.btn-danger {
+  background-color: var(--btn-danger-color);
+}
+
+.btn-warning {
+  background-color: var(--btn-warning-color);
+}
+
+.btn-default {
+  background-color: var(--primary-color);
+}
+
+.btn-link {
+  display: inline-block;
+  text-decoration: none;
+  text-align: center;
+  margin: 2rem;
+  padding: 15px 10px;
+  font-size: 1.8rem;
+  font-weight: 300;
+  border: none;
+  border-radius: 1rem;
+  color: #fafafa;
```

Comment:
It might be beneficial to create variable for this color, since it's been used in multiple places

Reply:
done